.env.example

```diff
@@ -1,22 +1,28 @@
-## Network chain ID to run your service on
+## Comma-separated list of chain ids of each network you want to run  your service on.
 ## Available IDS:      
 ## 31337 = local (hardhat)
 ## 5 = Goerli
 ## 11155111 = Sepolia
 ## 1 = Mainnet
 ## 80001 = Polygon Mumbai
 ## 84531 = Base Goerli
-CHAIN_ID=31337
+CHAIN_IDS=
 
 ## Key used to sign ETH transactions
-## Defaults below are hardhat deployer keys
 ETH_PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
 
 ## Mnemonic used to generate keys used to encrypt keyshares
 ENCRYPTION_MNEMONIC=
 
-## RPC provider. If running on local network, this env var is not required
-PROVIDER_URL=
+## Define RPC provider urls here. Local network provider url is not required.
+## Uncomment and setwhichever network chain id you want to run on.
+## At least one provider url is required.
+## ============================================================================
+# MAINNET_PROVIDER_URL=
+# GOERLI_PROVIDER_URL=
+# SEPOLIA_PROVIDER_URL=
+# POLYGON_MUMBAI_PROVIDER_URL=
+# BASE_GOERLI_PROVIDER_URL=
```

Comment:
This is missing the multiple encryption mnemonics

one thing I think we should consider while we are doing this large update, is changing the .env file to a json config file, more like how hardhat configs are setup. This may help with readability + organization, and we may also be able to force type safety (like numbers and strings)

```
networks: {
  sepolia: {
    chain_id: xxx,
    provider_url: xxx,
    mnemonic: xxx,
  },
  goerli: {
    chain_id: xxx,
    provider_url: xxx,
    mnemonic: xxx,
  }
}
```